idea/idea-core/src/org/jetbrains/kotlin/idea/core/psiModificationUtils.kt

```diff
@@ -354,11 +360,11 @@ fun KtTypeParameterListOwner.addTypeParameter(typeParameter: KtTypeParameter): K
     val list = KtPsiFactory(this).createTypeParameterList("<X>")
     list.parameters[0].replace(typeParameter)
     val leftAnchor = when (this) {
-        is KtClass -> nameIdentifier ?: getClassOrInterfaceKeyword()
-        is KtNamedFunction -> funKeyword
-        is KtProperty -> valOrVarKeyword
-        else -> null
-    } ?: return null
+                         is KtClass -> nameIdentifier ?: getClassOrInterfaceKeyword()
```

Comment:
There is a lot of unnecessary code formatting, which make harder to find what exactly author of PR changed. @Dimach, did was intentional and done by you or automatically changed by IntelliJ?